docs/guide/styles/standalone-usage.md

```diff
@@ -41,7 +41,7 @@ You can even **scope** the styles and/or variables to a container class to preve
 @use "sass:meta";
 
 // Import Kongponents Sass variables (path may vary)
-@import "@kong/kongponents/dist/_variables.scss";
+@import '@kong/kongponents/dist/_variables.scss';
```

Comment:
I think this whole section of the docs needs to be removed now, right?

Reply:
Yup pretty much. I was thinking about it and I suggest we create one story for docs and add a sub-ticket to go through all non-component related docs pages to make sure docs related to legacy functionality are removed (like this page). Writing migration guide could be another sub-ticket in that story.

Reply:
I already removed it - this is resolved; I'll add some docs tickets